app/controllers/shopify_app/fulfillment_services_controller.rb

```diff
@@ -0,0 +1,23 @@
+module ShopifyApp
+  class FulfillmentServicesController < ApplicationController
+    class ShopifyApp::MissingFulfillmentServiceError < StandardError; end
+
+    def action_missing(name)
+      render json: fulfillment_service_klass.send(name, fulfilment_service_params)
+    end
```

Comment:
why is this method called action_missing? shouldn't there be a method for each action or a method missing that tries to send it to the right method?